apps/condo/domains/miniapp/schema/plugins/serviceUserAccessForB2BApp.js

```diff
@@ -397,125 +361,13 @@ const addCustomAccessToSchema = (schemaName, schema, schemaConfig) => {
     }
 
     schema.access = wrapInCustomAccess(access, customListAccess)
```

Comment:
It's better to call the helper function inside Mode.access rather than use an implicit wrapper here.

Reply:
It is better to explicitly call this function inside the access check